README.md

```diff
@@ -0,0 +1,34 @@
+# BETTER BUILDS
+This React website is for users to submit and review PC builds that will be posted to PostgreSQL database. Users can also edit their reviews and vote on the reviews of other users with Express backend, running on a NodeJS environment.
+## Contributors: 
+Nick DeRis - github.com/dericholas
+Nick Kwiatkowski - github.com/nickkwiat
+Robert Le - github.com/fuzzyphantasm
+Corey Pierce - github.com/cornfrog
+Marlon Sanchez - github.com/MarlonS419 
+## Installation
+```
+git clone https://github.com/MarlonS419/better-builds
+```
+then Navigate to <localhost:3000/> in your browser
+## Development
+```
```

Comment:
same here, can designate the code block as shell. there are many formats that can be used in code blocks, for example adding js after the backticks will try to color the code block like javascript